BidnyiVitaliy/HW4/task1.py

```diff
@@ -0,0 +1,14 @@
+# Example output:
+# Enter the temperature in Celsius: 25
+# 25°C is equivalent to 77°F
+# Example output (if the user enters a temperature below absolute zero):
+# Enter the temperature in Celsius: -300
+# Error: Temperature below absolute zero (-273.15°C)
+
```

Comment:
good job
but try to use more meaningful naming of variables like this
celsius = float(input('Enter the temperature in Celsius: '))

if celsius < -273.15:
    print('Error: Temperature below absolute zero (-273.15°C)')
else:
    fahrenheit = celsius * 9/5 + 32
    print('{}°C is equivalent to {}°F'.format(celsius, fahrenheit))